docs/manifest.json

```diff
@@ -12,24 +12,33 @@
         { "title": "Known Issues", "path": "/docs/basics/known-issues.md" }
       ]
     },
+    {
+      "title": "Tips & Tricks",
+      "heading": true,
+      "routes": [
+	    { "title": "Basics", "path": "/docs/advanced/tips-and-tricks.md#basics" },
```

Comment:
Path should be without `advanced` folder, since we have `tips-and-tricks`  under `docs`